src/library/trackset/baseplaylistfeature.cpp

```diff
@@ -731,8 +738,8 @@ void BasePlaylistFeature::updateChildModel(int playlistId) {
 }
 
 /**
-  * Clears the child model dynamically, but the invisible root item remains
-  */
+ * Clears the child model dynamically, but the invisible root item remains
+ */
```

Comment:
We don't use Java style comments in new code.

Reply:
formatted in latest commit